jonasbb/python-snippets
# Add CI pipelines for the project

* Run pytest tests (none available yet)
* Run typechecking using mypy
* Check validity of GitHub actions and dependabot configuration
